host-controller/src/main/java/org/jboss/as/host/controller/operations/HttpManagementAddHandler.java

```diff
@@ -173,10 +176,7 @@ protected void installServices(OperationContext context, HttpInterfaceCommonPoli
             RemotingHttpUpgradeService.installServices(serviceTarget, ManagementRemotingServices.HTTP_CONNECTOR, httpConnectorName,
                     ManagementRemotingServices.MANAGEMENT_ENDPOINT, commonPolicy.getConnectorOptions());
         }
-    }
-
-    private ConsoleMode consoleMode(boolean consoleEnabled, boolean adminOnly) {
-        return consoleEnabled ? adminOnly ?  ConsoleMode.ADMIN_ONLY : ConsoleMode.CONSOLE : ConsoleMode.NO_CONSOLE;
+        return Arrays.asList(ManagementRemotingServices.MANAGEMENT_ENDPOINT, UndertowHttpManagementService.SERVICE_NAME);
```

Comment:
I don't believe anything related to ManagementRemotingServices is required unless http-upgrade is turned on.